test/integration/integration_admin_test.cc

```diff
@@ -552,20 +559,31 @@ TEST_P(IntegrationAdminTest, AdminDrainInboundOnlyGracefulConnectionCloseForInbo
   EXPECT_EQ("text/plain; charset=UTF-8", contentType(response));
   EXPECT_EQ("OK\n", response->body());
 
-  response = IntegrationUtil::makeSingleRequest(lookupPort("inbound_0"), "GET", "/", "",
-                                                downstreamProtocol(), version_);
-  EXPECT_TRUE(response->complete());
+  codec_client_ = makeHttpConnection(lookupPort("inbound_0"));
+  EXPECT_FALSE(codec_client_->disconnected());
+
+  IntegrationStreamDecoderPtr response4;
+  response4 = codec_client_->makeHeaderOnlyRequest(default_request_headers_);
```

Comment:
AFAICT this is second response for this test?

Reply:
Over broad rename; will fix